src/test/java/ru/levelup/elena/roshchina/qa/final_work/AbstractBaseFleetManagementTest.java

```diff
@@ -0,0 +1,83 @@
+package ru.levelup.elena.roshchina.qa.final_work;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+
+import java.io.*;
+import java.net.URL;
+import java.util.HashMap;
+
+
+public abstract class AbstractBaseFleetManagementTest {
+
+    protected WebDriver driver;
+    public HashMap<String, String> myCar = new HashMap<String, String>();
+
+    private final String carDataFileName = "car_data.csv";
+    private final String userLoginDataFileName = "login_data.csv";
+    private URL dataFilePathName = this.getClass().getResource("/ru.levelup.elena.roshchina.qa/data_files/final_work/");
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [UI: Usage of GetResource may be unsafe if class is extended (UI_INHERITANCE_UNSAFE_GETRESOURCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#ui-usage-of-getresource-may-be-unsafe-if-class-is-extended-ui-inheritance-unsafe-getresource).
Calling this.getClass().getResource(...) could give results other than expected if this class is extended by a class in another package.